packages/core/src/components/modal/modal.tsx

```diff
@@ -34,6 +34,7 @@ export class Modal implements ComponentInterface {
    * the role for the modal `dialog` | `alert` | `alertdialog`.
    */
   @Prop()
+  // eslint-disable-next-line @stencil-community/reserved-member-names
```

Comment:
Ah! Maar hier moeten we dus iets mee. En dit wordt een breaking change. Wil jij dit verwerken in #2447? Dit gaat nu over een andere Lint error, maar omdat dit ook om breaking changes gaat stel ik voor de scope te verruimen.

Reply:
Heb #2447 uitgebreid/aangepast